.github/workflows/build-pr.yml

```diff
@@ -0,0 +1,83 @@
+name: build-pr
+
+on:
+  push:
+    branches:
+      - develop
```

Comment:
This runs on pushes to `develop`, but doesn't remove the file. Will the upload overwrite this one based on the `ref` being the branch name (`develop`) over the commit hash?

If so that seems fine, but just wanna make sure this isn't going to spawn a lot of ZIP files 😅

Reply:
The generated zip files are renamed as `google-site-kit.zip` and `google-site-kit-dev.zip` so yes 😄

Reply:
note: the github.ref in this case is `refs/heads/develop` (so that is where the zips wind up being stored)